data/sql/updates/pending_db_world/juggling_torch_fix.sql

```diff
@@ -0,0 +1,3 @@
+DELETE FROM `spell_script_names` WHERE `spell_id` = '45644';
```

Comment:
```suggestion
DELETE FROM `spell_script_names` WHERE `spell_id` = 45644;
```
Please don't use ' around numbers unless the column data is string